frontend/lib/home/home_page.dart

```diff
@@ -43,12 +54,7 @@ class _HomePageState extends State<HomePage> {
         "Suche",
         GlobalKey<NavigatorState>(debugLabel: "Search tab key"),
       ),
-      AppFlow(
-        const IdentificationPage(title: "Ausweisen"),
-        Icons.remove_red_eye_outlined,
-        "Ausweisen",
-        GlobalKey<NavigatorState>(debugLabel: "Auth tab key"),
-      ),
+      ...additionalTabs,
```

Comment:
Can't you do an if clause inside the array declaration? 
Would be a lot easier to read.
https://github.com/dart-lang/language/blob/master/accepted/2.3/control-flow-collections/feature-specification.md

Reply:
I google for this feature but failed 😂